### PR TITLE
chore: removing beemo and updating components to use as prop

### PR DESCRIPTION
**Pull request by bclark-p44**
Closes # 

## 📝 Description

beemojs has been deprecated by the author. Leveraging turbo and tsup instead. Updated all components to use the as prop as I need to update each file because of parcel bundler errors with `__DEV__`.

## Screenshots

> Please provide screenshots for any visual changes

## Merge checklist

- [ ] Added/updated tests
- [x] Added changeset

### CODE REVIEW

File: packages/react/src/components/TextArea/TextArea.tsx [hunk 1]
```diff
@@ -26,63 +24,58 @@ interface TextAreaProps extends Omit<TextFieldBaseProps, 'endIcon' | 'startIcon'
...
-  const [inputValue, setInputValue] = useControlledState(props.value, props.defaultValue, () => {
-    //noop
-  });
+    const areaRef = React.useRef<HTMLTextAreaElement>(null);
 
-  const handleHeightChange = React.useCallback(() => {
-    const input = areaRef.current as HTMLTextAreaElement;
-    const prevAlignment = input.style.alignSelf;
+    const [inputValue, setInputValue] = useControlledState(props.value, props.defaultValue, () => {
+      //noop
```
Comment (Ashley-p44) [inline packages/react/src/components/TextArea/TextArea.tsx:42]: did you mean to leave this comment?

**Review by Ashley-p44 — APPROVED** [review]
Bye bye beemo 👋🏻 .

File: packages/react/src/components/TextArea/TextArea.tsx (hunk 1 above)
Comment (bclark-p44) [reply packages/react/src/components/TextArea/TextArea.tsx:42]: yep, react aria expects a default function and eslint complains about empty functions. this was a way around it

File: packages/react/src/index.ts [hunk 2]
```diff
@@ -1,48 +1,150 @@
...
-export * from './components/TableFooter';
-export * from './components/TableHeader';
-export * from './components/TableRow';
-export * from './components/Tag';
-export * from './components/TextArea';
-export * from './components/TextField';
-export * from './components/Tooltip';
-export * from './components/Typography';
-export * from './components/VisuallyHidden';
+export { Avatar } from './components/Avatar';
```
Comment (bclark-p44) [inline packages/react/src/index.ts:1]: Making exports more explicit to make sure nothing we dont want to expose slips through

**Pull request merged**